### DISCUSSION TITLE
IM release not accessible anymore

### DISCUSSION THREAD

**Discussion by blubberblah** [ImageMagick/ImageMagick#4407] [General] [answered]
In our buildprocess we refer to files located at: https://download.imagemagick.org/ImageMagick/download/releases/
It happend several times that files got removed which resulted in breaking our builds. Actually it happened again for https://download.imagemagick.org/ImageMagick/download/releases/ImageMagick-7.1.0-3.tar.gz
Could you please tell me if there are more stable Links to refer to?
Regards

**Comment by dlemstra** [comment, marked as answer]
We cleanup our downloads to save space. You could use an "unofficial" download from one of our tags. For example: https://github.com/ImageMagick/ImageMagick/releases/tag/7.1.0-11.

**Reply by blubberblah** [reply]
Thanks for the quick reply :-)
